remoulade/api/main.py

```diff
@@ -76,6 +86,23 @@ def get_actors():
     return {"result": [actor.as_dict() for actor in get_broker().actors.values()]}
 
 
+@app.route("/groups")
+def get_groups():
+    args = PageSchema().load(request.args.to_dict())
+    backend = remoulade.get_broker().get_state_backend()
+    groups = defaultdict(list)
+    states = backend.get_states()
+    if args.get("search_value"):
```

Comment:
you check twice for group_id. maybe it would be simpler to do:
`states = (state for state in backend.get_states() if state.group_id)`